jscomp/frontend/external_ffi_types.ml

```diff
@@ -300,6 +300,9 @@ let inline_int_primitive (i : int32) : string list =
 let inline_int64_primitive (i : int64) : string list =
   [""; to_string (Ffi_inline_const (Const_int64 i))]
 
+let inline_bigint_primitive (i : string) : string list =
+  [""; to_string (Ffi_inline_const (Const_bigint i))]
```

Comment:
This does not check that the bigint is valid, or remove leading zero's.
Don't know if this is a problem, just observing the fact.

Reply:
One possibility is to do the check `is_numeric` at parsing time, instead of type checking time. And do the normalisation (remove zeros) whenever a `Const_bigint` is constructed.

Reply:
Very good. I'm going to try to refactor `Const_bigint bool * string` in AST type.